inst/qml/TTestPairedSamples.qml

```diff
@@ -29,7 +29,7 @@ Form
 	plotHeight: 300
 	plotWidth:  350
 
-	CheckBox { name: "welchs"; checked: false; visible: false }
+	CheckBox { name: "welch"; checked: false; visible: false }
```

Comment:
do you know what is the purpose of this option in the first place? It seems like it does not do anything? Can we remove it?